src/v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -43,7 +44,7 @@ const hashEncrypt = (object) => {
  * @param {*} param2
  * @returns
  */
-const responseBuilder = (metadata, body, { Config }, message) => {
+const responseBuilder = (metadata, body, { Config }, message, consentBlock) => {
```

Comment:
did we make sure to check all the calls to this function?

Reply:
yes, this function is being called only once. Also passing the whole params now to avoid missing of variables